src/main/kotlin/com/terraformation/backend/accelerator/ParticipantProjectSpeciesService.kt

```diff
@@ -0,0 +1,45 @@
+package com.terraformation.backend.accelerator
+
+import com.terraformation.backend.accelerator.db.ParticipantProjectSpeciesStore
+import com.terraformation.backend.accelerator.db.SubmissionStore
+import com.terraformation.backend.accelerator.model.ExistingParticipantProjectSpeciesModel
+import com.terraformation.backend.accelerator.model.NewParticipantProjectSpeciesModel
+import com.terraformation.backend.db.default_schema.ProjectId
+import com.terraformation.backend.db.default_schema.SpeciesId
+import jakarta.inject.Named
+
+@Named
+class ParticipantProjectSpeciesService(
+    private val participantProjectSpeciesStore: ParticipantProjectSpeciesStore,
+    private val submissionStore: SubmissionStore,
+) {
+  /** Creates a new participant project species, possibly creating a deliverable submission. */
+  fun create(model: NewParticipantProjectSpeciesModel): ExistingParticipantProjectSpeciesModel {
+    val existingModel = participantProjectSpeciesStore.create(model)
+
+    // If a submission doesn't exist for the deliverable, create one
+    val deliverableSubmission =
+        submissionStore.fetchActiveSpeciesDeliverableSubmission(model.projectId)
+    if (deliverableSubmission.submissionId == null) {
+      submissionStore.createSubmission(deliverableSubmission.deliverableId, model.projectId)
+    }
```

Comment:
Wrapping this (and the other one) in a transaction would probably be good so you don't have a dangling participant project species if creating the submission fails for some reason.